src/components/StatusPanel.tsx

```diff
@@ -71,6 +71,8 @@ export const StatusPanel: React.FC<Props> = ({
     ? 'noData'
     : 'ok';
 
+
```

Comment:
Not something that we need to do in this PR but in the long run we should probably construct the css styles in a memoized function instead of doing it inline during every re-render.

https://github.com/grafana/grafana/blob/c46c0c9eb9ac91a84dd425917327945cf19daa35/contribute/style-guides/themes.md?plain=1#L18